src/main/java/mocacong/server/domain/Member.java

```diff
@@ -101,7 +106,28 @@ private void validatePhone(String phone) {
         }
     }
 
+    private void validatePassword(String password) {
+        if (!PASSWORD_REGEX.matcher(password).matches()) {
+            throw new InvalidPasswordException();
+        }
+    }
+
+    public void validateUpdateInfo(String nickname, String password, String phone) {
+        if (StringUtils.isBlank(nickname) || StringUtils.isBlank(password) || StringUtils.isBlank(phone)) {
+            throw new InvalidArgumentException();
+        }
+        validateNickname(nickname);
+        validatePassword(password);
+        validatePhone(phone);
+    }
+
     public void updateProfileImgUrl(String imgUrl) {
         this.imgUrl = imgUrl;
     }
+
+    public void updateProfileInfo(String nickname, String password, String phone) {
+        this.nickname = nickname;
+        this.password = password;
+        this.phone = phone;
+    }
```

Comment:
```suggestion
    public void updateProfileInfo(String nickname, String password, String phone) {
        validateMemberInfo(nickname, phone);
        this.nickname = nickname;
        this.password = password;
        this.phone = phone;
    }
```
여기서만 검증하고, 추가된 다른 도메인 로직은 지워도 괜찮아보이네요.
추가로, public method가 private method보다 위쪽에 있는 게 가독성이 좋을 거 같아요

Reply:
넵 이 부분 유의하겠습니다.